src/exojax/spec/molinfo.py

```diff
@@ -27,63 +42,182 @@ def molmass(molecule, db_HIT=False):
        >>> print(molmass("air"))
        >>> 28.97
     """
-    dt = pkgutil.get_data('exojax', 'data/atom/HITRAN_molparam.txt')
-    df = pd.read_csv(BytesIO(dt), sep="\s{2,}", engine="python", skiprows=1, \
-                     names=["# Iso","Abundance","Q(296K)","gj","Molar Mass(g)"])
-    molmass_hi = {}
-    for i in range(len(df)):
-        if("(" in df["# Iso"][i]):
-            molname = df["# Iso"][i].split()[0]
-            tot = 0.0
-            tot_abd = 0.0
-        else:
-            tot = tot + df["Abundance"][i] * df["Molar Mass(g)"][i]
-            tot_abd = tot_abd + df["Abundance"][i]
+    molmass_isotope, abundance_isotope = molmass_hitran()
 
-        if(i == len(df)-1 or "(" in df["# Iso"][i+1]):
-            molmass_hi[molname] = tot/tot_abd
-
-
-    if molecule == 'air' or molecule == 'Air':
+    if simple_molecule_name == 'air' or simple_molecule_name == 'Air':
         return 28.97
 
-    if molecule in molmass_hi and db_HIT:
-        molmass = molmass_hi[molecule]
+    if simple_molecule_name in molmass_isotope and db_HIT:
+        molmass = molmass_isotope[simple_molecule_name][0]
```

Comment:
If you want to extract the mass for the most abundant isotope combination, I think this line should be
```suggestion
        molmass = molmass_isotope[simple_molecule_name][1]
```
since [0] corresponds to the mean molar mass as written [here](https://github.com/HajimeKawahara/exojax/blob/e1745de3736c1c469b58297f4c7ca56561fbb12f/src/exojax/utils/isotopes.py#L82).
I suppose this is why I currently get different results from `molinfo.molmass_isotope("CO")` and `molinfo.isotope_molmass("12C-16O")`.